articles/load-balancer/outbound-rules.md

```diff
@@ -31,7 +31,7 @@ With outbound rules, you can explicitly define outbound **SNAT** behavior.
 Outbound rules allow you to control:
 
 * **Which virtual machines are translated to which public IP addresses.**
-     * Two rules were backend pool 1 uses the blue IP address 1 and 2, backend pool 2 uses the yellow IP prefix.
+     * Two rules where backend pool 1 uses the blue IP address 1 and 2, backend pool 2 uses the yellow IP prefix.
```

Comment:
This sentence is unclear. Does either of the following revisions work in its place?

     * Two rules where backend pool 1 uses the blue IP address 1 and 2, and backend pool 2 uses the yellow IP prefix.

OR

   * Two rules where backend pool 1 uses the blue IP address 1 and backend pool 2 uses the yellow IP prefix.